spec/rubocop/cop/rails/read_write_attribute_spec.rb

```diff
@@ -16,4 +16,87 @@
     expect(cop.offenses.size).to eq(1)
     expect(cop.highlights).to eq(['write_attribute'])
   end
+
+  describe '#autocorrect' do
+    context 'write_attribute' do
```

Comment:
What if there's an explicit receiver, `object.write_attribute(:attr, var)`? Leaving out the parentheses in one of the examples would also be good.